public/api/conf/1.0/sandbox/get-state-pension.md

```diff
@@ -8,89 +8,89 @@
         </tr>
         <tr>
-            <td><p>Exclusion</p></td>
+            <td><p>Taxpayer with the IsleOfMan exclusion</p></td>
             <td><p><code>MA</code></p></td>
-            <td><p>A user with the IsleOfMan exclusion</p></td>
+            <td><p>200 (OK) Payload as per exclusion example above</p></td>
         </tr>
         <tr>
-            <td><p>Exclusion</p></td>
+            <td><p>Taxpayer with the MarriedWomenReducedRateElection exclusion</p></td>
             <td><p><code>MW</code></p></td>
-            <td><p>A user with the MarriedWomenReducedRateElection exclusion</p></td>
+            <td><p>200 (OK) Payload as per exclusion example above</p></td>
         </tr>
         <tr>
-            <td><p>Exclusion</p></td>
+            <td><p>Taxpayer that is excluded for multiple reasons</p></td>
             <td><p><code>MX</code></p></td>
-            <td><p>A user that is excluded for multiple reasons</p></td>
+            <td><p>200 (OK) Payload as per exclusion example above</p></td>
         </tr>
         <tr>
-            <td><p>Success</p></td>
+            <td><p>Taxpayer with COPE</p></td>
             <td><p><code>CR</code></p></td>
-            <td><p>A user with COPE</p></td>
+            <td><p>200 (OK) Payload as per regular example above</p></td>
         </tr>
         <tr>
-            <td><p>Success</p></td>
+            <td><p>Taxpayer with a protected payment</p></td>
             <td><p><code>PP</code></p></td>
-            <td><p>A user with a protected payment</p></td>
+            <td><p>200 (OK) Payload as per regular example above</p></td>
         </tr>
         <tr>
-            <td><p>Success</p></td>
+            <td><p>Taxpayer below the minimum qualifying period who cannot get any State Pension even by working until State Pension age and filling payable gaps</p></td>
             <td><p><code>CA</code></p></td>
-            <td><p>A user below the minimum qualifying period who cannot get any State Pension even by working until State Pension age and filling payable gaps</p></td>
+            <td><p>200 (OK) Payload as per regular example above</p></td>
         </tr>
         <tr>
-            <td><p>Success</p></td>
+            <td><p>Taxpayer below the minimum qualifying period who can get some State Pension by working until State Pension age and also filling payable gaps</p></td>
             <td><p><code>AL</code></p></td>
-            <td><p>A user below the minimum qualifying period who can get some State Pension by working until State Pension age and also filling payable gaps</p></td>
+            <td><p>200 (OK) Payload as per regular example above</p></td>
         </tr>
         <tr>
-            <td><p>Success</p></td>
+            <td><p>Taxpayer below the minimum qualifying period that can get some State Pension by just working to State Pension age, but can get more by filling payable gaps as well</p></td>
```

Comment:
Taxpayer below the minimum qualifying period that can get some State Pension by working until State Pension age, but can get more by filling payable gaps as well